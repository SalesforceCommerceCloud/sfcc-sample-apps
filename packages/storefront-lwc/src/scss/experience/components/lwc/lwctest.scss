.lwctest {
    color: chocolate;
    background-color: azure;
}
