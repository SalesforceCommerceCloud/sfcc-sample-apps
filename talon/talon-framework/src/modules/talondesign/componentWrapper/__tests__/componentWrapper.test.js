import { createElement } from 'lwc';
import ComponentWrapper from 'talondesign/componentWrapper';

jest.mock('talon/componentService', () => ({
    async createElement(name) {
        return global.document.createElement(name);
    }
}));

describe('talondesign/componentWrapper', () => {
    it('renders', async () => {
        const element = createElement('talondesign-component-wrapper', { is: ComponentWrapper });
        element.componentId = "blah-blah-blah";
        element.label = "My Label!";
        element.name = "foo";
        element.isLocked = true;

        document.body.appendChild(element);
        expect(element).toMatchSnapshot();
    });
});