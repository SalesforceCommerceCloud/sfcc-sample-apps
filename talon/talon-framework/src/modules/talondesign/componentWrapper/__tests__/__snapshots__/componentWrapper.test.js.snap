// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`talondesign/componentWrapper renders 1`] = `
<talondesign-component-wrapper>
  #shadow-root(open)
    <div
      class="ui-widget ui-interactions-component"
      data-allow-delete="true"
      data-allow-move="true"
      data-allow-select="true"
      data-component-label="My Label!"
      data-fqn="foo"
      data-item-id="blah-blah-blah"
    >
      <div
        class="actualNode"
      />
      <div
        class="elementProxyContainer"
      />
    </div>
</talondesign-component-wrapper>
`;
